test/common/stats/thread_local_store_test.cc

```diff
@@ -1564,6 +1566,22 @@ TEST_F(HistogramTest, ParentHistogramBucketSummary) {
             "B3.6e+06(1,1)",
             parent_histogram->bucketSummary());
 }
+
+class SinkPredicatesImpl : public SinkPredicates {
```

Comment:
for consistency with other tests I'd call this class `TestSinkPredicates` to indicate this is a test version and not the primary impl.